src/app/functions/server/reactivateAccount.ts

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { SubscriberRow } from "knex/types/tables";
+import { logger } from "./logging";
+import { reactivate } from "../../../utils/fxa";
+import { record } from "./glean";
+
+export async function reactivateAccount(subscriber: SubscriberRow) {
+  logger.info("fxa_reactivate_user", {
+    subscriber: subscriber.id,
+  });
+
+  record("account", "reactivate", {
```

Comment:
@rhelmer I'm not sure if Glean is set up here for this event. If not we can take it out?

Reply:
We're currently logging it (so it appears in log console and BQ), but I don't think Glean is set up to ingest it into the main telemetry BQ. Feel free to remove if you need to.